IBX-702: Added missing note about required normalizer tag in Migrations (#1407)
ezsystems/developer-documentation: docs/guide/data_migration.md

```

> Then the action denormalizer needs to be tagged to be recognized by serializer used for migrations.

Reviewer: ```suggestion
Then, tag the action denormalizer so it is recognized by the serializer used for migrations.
```

``` yaml
services:
    Ibexa\Platform\Bundle\Migration\Serializer\Denormalizer\ContentType\Action\AssignContentTypeGroupActionDenormalizer:

>         # Autoconfigure set to false will prevent this normalizer from being registered in your application's serializer

Reviewer: ```suggestion
        # When autoconfigure is set to false, this normalizer is not registered in your application's serializer
```

Author: True, but this is practically a warning - it's something the user **should not do** (setting `autoconfigure` to `true` in this service).

Usually it should not affect much, although it makes little sense to add Migration normalizers to the default application normalizer.

Reviewer: Ok, how about: `Make sure to set autoconfigure to false, otherwise...`?

Author: `Make sure to set autoconfigure to false, otherwise this normalizer will be registered in your application's main serializer`?

Reviewer: Yep, sounds good
